locales/en-US/notes.properties

```diff
@@ -95,7 +95,7 @@ title1=Heading 1
 title2=Heading 2
 title3=Heading 3
 paragraph=Body
-bodyText=Body text
+bodyText=Body Text
```

Comment:
I guess we cannot really change any of this text like. every change needs to introduce a new variable. However these are minimal changes to English that won't matter that much for other locales? @flodolo let us know

Reply:
What about using CSS like `text-transform: capitalize` on bodyText ? Would avoid a review from i18n team and force styling for every language.

Reply:
Let's keep the existing string IDs, each locale should have their own rules for capitalization.

Using CSS text-transform is a no-go for web projects, where we don't control the rendering engine. 

If the only engine is Gecko, we know it does the right thing with `uppercase`, as long as you pass the correct language code. For the record, that causes bugs in Firefox UI.

In general, it's better to use capitalization in the string and let each locale decide what to use.

`capitalize` is really poor, since it's not proper letter case even in English.